queries/sflog/highlights.scm

```diff
@@ -0,0 +1,44 @@
+; highlights.scm
+(number) @number
+
+(identifier) @variable
+
+(version) @string.special
+
+(namespace) @variable
+
+(limit) @property
+
+(limit
+  (identifier) @string)
+
+(limit
+  (number) @number)
+
+(duration
+  (number) @number) @string
```

Comment:
I think this whole query should be removed: number is already covered and other than that we just have a few parenthesis which should eventually be covered as `@punctuation.bracket`